app/jobs/github_import_job.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'github_importer'
+
+# @abstract an abstract class for importing files from a Github directory
+class GithubImportJob < ApplicationJob
+  queue_as :default
+
+  def perform(github_token)
```

Comment:
Maybe this is a good time to fully transition this to `Settings.import.access_token`?